Vagrantfile

```diff
@@ -1,7 +1,9 @@
 
 $script = <<SCRIPT
+sudo add-apt-repository ppa:ondrej/php
```

Comment:
I think Ubuntu Xenial have PHP 7 in the standard packages. The PPA should not be necessary, what about using php7?